internal/binutils/testdata/build_binaries.go

```diff
@@ -70,6 +70,7 @@ func main() {
 		if err != nil {
 			log.Fatal(err)
 		}
+		log.Println("Please verify that exe_windows_64.exe does not trigger any anti-virus.")
```

Comment:
A hint on how to do the check would be good.

Also, can we document the recommended build environment?

Reply:
I was wondering whether I should include a link to virustotal.com, it could get outdated for some reason. But, I can add that documentation.

Sure, I'll add a link to how to install MSYS2, however, similar concerns how long those links will work.

Reply:
Done.